Functions.Templates/Templates/DurableFunctionsEntity-Python/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Should this be in 1.x bundle or 2.x or both?

Reply:
only 2.x